config/channels/in-memory-channel/200-imc-dispatcher-serviceaccount.yaml

```diff
@@ -65,3 +65,9 @@ rules:
     - get
     - list
     - watch
+- apiGroups:
+    - ""
+  resources:
+    - "serviceaccounts/token"
+  verbs:
+    - create
```

Comment:
This seems to be the wrong place. There is also a [clusterrole](https://github.com/knative/eventing/blob/main/config/channels/in-memory-channel/roles/dispatcher-clusterrole.yaml) for the imc-dispatcher